proposals/lip-0043.md

```diff
@@ -163,17 +125,21 @@ sidechainRegParams = {
     "type": "object",
     "required": [
         "name",
-        "genesisBlockID",
+        "chainID",
         "initValidators",
-        "certificateThreshold"
+        "certificateThreshold",
+        "sidechainRegistrationFee"
     ],
     "properties": {
         "name": {
             "dataType": "string",
+            "minLength": MIN_CHAIN_NAME_LENGTH,
```

Comment:
These constants are not defined in LIP 45, as far as I can see.

Reply:
yes, I still have to finalize that LIP

Reply:
Resolve the conversation if you don't need it as a reminder.